gatherers/censys.py

```diff
@@ -79,7 +79,10 @@ def paginated_mode(suffix, options, uid, api_key):
 
     certificate_api = certificates.CensysCertificates(uid, api_key)
 
-    query = "parsed.subject.common_name:\"%s\" or parsed.extensions.subject_alt_name.dns_names:\"%s\"" % (suffix, suffix)
+    if 'query' in options and options['query']:
```

Comment:
A small nit - this can be `if options.get('query'):` (it's a safe function call, and will return `None` if `query` isn't a key in the dict). But I can make that change myself, no reason to hold up the query.